MiniF/.pylintrc

```diff
@@ -0,0 +1,48 @@
+[MASTER]
+# Django plugin + ігнор згенерованого/непотрібного
+load-plugins = pylint_django
+ignore = migrations,venv,node_modules,static
+jobs = 0
+
+# Ініціалізація Django, щоб уникнути E5110 (django-not-configured)
+init-hook =
+    import os, sys; \
+    sys.path.append(os.getcwd()); \
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE','MiniF.settings'); \
+    import django; django.setup()
+
+[MESSAGES CONTROL]
+# Корисне ядро (стиль + реальні помилки/якість)
+enable = E0602,W0612,C0103,C0301,R0913,R0914,R0915,E1101
+
+disable =
+    R0903,        # too-few-public-methods
+    C0114,        # missing-module-docstring
+    C0115,        # missing-class-docstring
```

Comment:
missing-class-docstring 

 I propose to make a decision on this item at a general meeting.